week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,22 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function percentage(number) {
+  return Math.round(number);
+}
+
+function message(numberOfStudents, numberOfMentors) {
+  var total = numberOfStudents + numberOfMentors;
+  var studentsPercentage = (100 * numberOfStudents) / total;
+  var mentorsPercentage = (100 * numberOfMentors) / total;
```

Comment:
Similarly to your code in `I-floats/exercise.js`, can you use another function to reduce duplicate code?

Reply:
I will use a function in `I-floats/exercise.js` as well.

Reply:
DRY implemented by using a function for displaying the message on the screen and a nested function for rounding the percentage.

Reply:
also duplicated code removed.